inc/composer/class-docker-compose-generator.php

```diff
@@ -237,6 +237,53 @@ protected function get_service_php() : array {
 		];
 	}
 
+	/**
+	 * Get the NodeJS container service.
+	 *
+	 * @return array
+	 */
+	protected function get_service_nodejs() : array {
+		$config = $this->get_config();
+
+		$version = (string) $this->get_config()['nodejs']['version'] ?? '20';
+
+		return [
+			'nodejs' => [
+				'image' => "node:{$version}-bookworm",
```

Comment:
Could we also set env vars for atleast `ALTIS_ENVIRONMENT_NAME` and `ALTIS_ENVIRONMENT_TYPE`, so people can do checks for `local`?

Reply:
Implemented